source/language/directives.rst

```diff
@@ -1,14 +1,86 @@
 Directives
 ==========
 
-OpenQASM supports a directive mechanism that allows other information to
-be included in the program. A directive begins with ``#pragma`` and
-terminates at the end of the line. Directives can provide annotations
-that give additional information to compiler passes and the target
-system or simulator. Ideally the meaning of the program does not change
+OpenQASM supports directive mechanisms that allows other information to
+be included in the program. Directives are either pragmas or annotations.
+Both are used to supply additional information to the compiler passes and the
+target system or simulator. Ideally the meaning of the program does not change
 if some or all of the directives are ignored, so they can be interpreted
 at the discretion of the consuming process.
 
+Pragmas
+-------
+
+Pragma directives start with ``pragma`` and continue to the end of line. The
+``#pragma`` syntax is also accepted for reverse compatibility. Pragmas should be
```

Comment:
```suggestion
``#pragma`` syntax is also accepted for backwards compatibility. Pragmas should be
```
I wonder if we should include this officially? I understand the AWS release has this, but since this is already a specification divergence, perhaps it is ok to keep it as such and begin returning warnings to users to switch to the specification syntax?